craft_application/application.py

```diff
@@ -53,7 +53,7 @@
         "craft_parts",
         "craft_providers",
         "craft_store",
-        "craft_application.remote",
+        "craft_application",
```

Comment:
How this affect logs generated via https://github.com/canonical/craft-application/blob/main/craft_application/git/_git_repo.py ?

Reply:
it affects them in the sense that now they'll be properly captured ;)